packages/cli/src/config/settings.ts

```diff
@@ -56,6 +56,12 @@ export interface AccessibilitySettings {
   disableLoadingPhrases?: boolean;
 }
 
+export interface PromptCompletionSettings {
+  enabled?: boolean;
+  minLength?: number;
+  debounceMs?: number;
```

Comment:
does debounceMs really need to be configurable in settings? I would suggest just using a reasonable default. same for minLength. I'd suggest we only have hard coded option letting you just use a simple boolean setting which is the easiest to implement correctly. otherwise you have to deal with merging and complexities surfacing this in the settings UI dialog which has now landed.